.github/workflows/publish-containers.yml

```diff
@@ -4,7 +4,7 @@ on:
   push:
     tags:
       # Matches stable and pre-releases
-      - "[0-9]+.[0-9]+.[0-9]+*"
+      - "v?[0-9]+.[0-9]+.[0-9]+*"
```

Comment:
It will change the tag format convention for container images from `1.2.3` to `v1.2.3`, is this the wanted behavior?